tensorflow.md

```diff
@@ -1,32 +1,77 @@
-# Building TensorFlow on Graviton2
+# ML inference on Graviton CPUs with TensorFlow
 
-Install bazel build system:
+There are two ways to use tensforflow based ml inferencing on Graviton CPUs.
+- Tensorflow DLAMI
+- Tensorflow Docker container
+
+**Using AWS DLAMI:**
+To launch a Graviton instance with a DLAMI via EC2 console:
+
+1. Click on "Community AMIs"
+2. Filter AMIs by clicking Ubuntu and Arm (64-bit) check boxes
+3. Search for "Deep Learning" in search field to list the current Ubuntu based DLAMIs"
+4. Select the AMI: "Deep Learning AMI Graviton Tensorflow <version>(Ubuntu 20.04) <yyyy/mm/dd>"
+
+Here is the awscli snippet to launch the same:
+
+aws ec2 run-instances --region us-west-2 --image-id ami-0e273fef68b695319  --instance-type c6g.4xlarge --count 1  --key-name <key_name_to_added> --subnet-id subnet-1bafd750 --block-device-mappings 'DeviceName=/dev/sda1,Ebs={VolumeSize=256}' --tag-specifications 'ResourceType=volume,Tags=[{Key=Name,Value="Graviton TF DLAMI"},{Key=< >,Value=< >}]' 'ResourceType=instance,Tags=[{Key=Name,Value="Graviton TF DLAMI"},{Key=< >,Value=< >}]'
+
+Once Graviton instance is launched with the above DLAMI, the platform is ready for building Tensorflow based ML inference applications.The DLAMI comes with the additional libraries preinstalled, eg: rust compilers, transformers etc, to enable transformers based (eg: bert) inferencing applications.
 ```
-# find the latest release of bazel for arm64 https://github.com/bazelbuild/bazel/releases/latest
-# using older bazel versions may not be supported to build tensorflow
-REL=4.2.1  # this is the latest stable release as of this writing
-wget https://github.com/bazelbuild/bazel/releases/download/${REL}/bazel-${REL}-linux-arm64
-chmod +x bazel-${REL}-linux-arm64
-sudo mv bazel-${REL}-linux-arm64 /usr/local/bin
-sudo ln -s /usr/local/bin/bazel-${REL}-linux-arm64 /usr/local/bin/bazel
+# One can check the TF version as below
+pip show tensorflow
+
+# By default eigen backend is enabled, it is recommended to switch to the optimized onednn+acl(arm compute library). This can be enabled by setting the below TF environment variable
```

Comment:
By default it's eigen, but typically onednn+acl provides better performance and this can be enabled by...